lib/node_modules/@stdlib/math/base/special/ellipj/docs/types/index.d.ts

```diff
@@ -0,0 +1,72 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 2.0
+
+/**
+* Computes the Jacobi elliptic functions sn, cn, and dn.
+*
+* ## Notes
+*
+* -   The functions are evaluated using the [complete elliptic integral of the first kind](https://en.wikipedia.org/wiki/Elliptic_integral#Complete_elliptic_integral_of_the_first_kind) `K`.
+*
+* -   The `x` argument is converted to double-precision floating-point format.
+*
+* -   The returned values are exact for `m` values where `|m| < 2**-24`.
+*
+* -   The functions return `NaN` for `m >= 1`.
+*
+* -   When `m < 1`, the following relations hold
+*
+*     ```tex
+*     \operatorname{sn}(x+x) = 2\operatorname{sn}(x)\operatorname{cn}(x)
+*     \operatorname{cn}(x+x) = 1 - 2\operatorname{sn}(x)^{2}
+*     \operatorname{dn}(x+x) = 1 - 2\operatorname{sn}(x)^{2}\operatorname{dn}(x)
+*     ```
+*
+* @param m - parameter
+* @param x - argument
+* @returns array containing three elements corresponding to the Jacobi elliptic functions
```

Comment:
I believe this is out-of-date. IIRC, the function returns the amplitude, as well.

We also need to document each of the public APIs.

Reply:
Not sure how familiar you are with TypeScript, so, if it would be easier, I can work on updating the declarations. Similar to other public APIs, we'll want to define an interface and document the methods individually.

Reply:
Ack! I always miss some of these docs. No need to submit a bunch of corrections for these; I'll work through these jsdoc annotations and try to make sure I didn't miss more.

Reply:
Ah, that's correct. I think this is actually just left over from the initial scaffolding.